app/views/layouts/application.html.slim

```diff
@@ -2,15 +2,17 @@ doctype html
 html
   head
     title Bookanook
+    meta name="Timezone" content=Time.zone
     = stylesheet_link_tag '//fonts.googleapis.com/css?family=Open+Sans:400,700'
     = stylesheet_link_tag  'application', media: 'all'
     = javascript_include_tag '//maps.google.com/maps/api/js?v=3.23&key=' + ENV["MAPS_API_KEY"]
     = javascript_include_tag '//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js'
+    = javascript_include_tag '//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.6/jstz.min.js'
```

Comment:
Can you download these and add them to the app, and then include them in the asset pipeline? We don't expect this app to need to scale such that a CDN is giving us much, and it's adding an external dependency we don't need. It's probably worth moving all three of these js libs into sprockets.